doc/adr/0007-deprecate-servicesync.md

```diff
@@ -0,0 +1,48 @@
+# 7. Deprecate ServiceSync
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+Enabled By [6. Explicit resources for declaring service bindings](0006-explicit-resources-for-declaring-service-bindings.md)
+
+## Context
+
+ServiceSync is a mechanism whereby service ingress bindings are
+automatically created at each site in the network for any service with
+egress bindings.
+
+Though this feature seemed nice in early demos, in practice it is
+generally unnecessary. In some cases it is explicitly not desirable as
+the ingress bindings should only be created in particular sites.
+
+Though the feature can at present be turned off, the option to do so
+is not obvious. Further it is not well tested.
+
+If the decision to have explicit resources for ingress bindings is
+adopted, it becomes even simpler to explicitly add the required
+ingresses to any site that requires them. Indeed as that proposal
+allows more flexibility with regard to the ingress bindings
+ServiceSync would not be adequate as it is.
+
+## Decision
+
+ServiceSync will not be supported for automatic propoagation of
```

Comment:
clarify "will not be supported" (e.g. will be removed) as the consequence below indicates re-introducing as a new/separate "van service"

Reply:
Done.